modules/030-cloud-provider-aws/docs/USAGE.md

```diff
@@ -2,7 +2,9 @@
 title: "Cloud provider — AWS: usage"
 ---
 
-## An example of the `AWSInstanceClass` CR
+## Custom resource `AWSInstanceClass`
```

Comment:
@Zhbert I don't agree with that. It is not a CR it is only one example. The CR is on the CR.html page.

Reply:
Maybe just decrypt “CR” in the title? 

And I'm not sure about the inscription below. But if we remove it, it will look a little empty, and earlier we decided to place it on the previous pages.